.evergreen/config.yml

```diff
@@ -2519,6 +2555,13 @@ buildvariants:
     - name: "aws-auth-test-4.4"
     - name: "aws-auth-test-latest"
 
+- matrix_name: "Release"
+  matrix_spec:
+    platform: [ubuntu-16.04, windows-64-vsMulti-small]
```

Comment:
Why not Ubuntu 18? 16 goes EOL in April.

Reply:
Changed to 18. They both have Docker so either one works.